webpack/components/EditableTextInput/EditableTextInput.js

```diff
@@ -110,10 +110,12 @@ EditableTextInput.propTypes = {
   value: PropTypes.string,
   attribute: PropTypes.string.isRequired,
   textArea: PropTypes.bool, // Is a text area instead of input when editing
+  placeholder: PropTypes.string,
 };
 
 EditableTextInput.defaultProps = {
   textArea: false,
+  placeholder: '',
```

Comment:
If you add default value as __('None provided'), we can just do (<i>{placeholder}</i> above..

Reply:
I like.